index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <link href="css/style.css" rel="stylesheet" type="text/css" />
+        <title>Apple</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    </head>
+    <body>
+        <header class="header row">
+            <nav>
+                <ul>
+                    <li class="logo"><a href="#" ><img src="img/logo.png" alt="apple-logo"></a></li>
+                    <li><a href="#">Store</a></li>
+                    <li><a href="#">Mac</a></li>
+                    <li><a href="#">iPod</a></li>
+                    <li><a href="#">iPhone</a></li>
+                    <li><a href="#">iPad</a></li>
+                    <li><a href="#">iTunes</a></li>
+                    <li><a href="#">Support</a></li>
+                    <li><input type="search" /></li>
+                </ul>
+            </nav>
```

Comment:
**[OPTIONAL]** Close the header tag here and open the main tag after it that will hold the main section, and use position absolute for the header tag so you make it overflow the main section. in order to make your code more semantic